tools/data/utils/img2lmdb.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import json
+import os
+import os.path as osp
+
+import cv2
+import lmdb
+import numpy as np
+
+
+def checkImageIsValid(imageBin):
```

Comment:
```suggestion
def check_image_is_valid(image_bin):
```
Use snake case instead of camel case